apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/ListJobsOptions.cs

```diff
@@ -39,6 +39,11 @@ public sealed class ListJobsOptions
         /// </summary>
         public bool? AllUsers { get; set; }
 
+        /// <summary>
+        /// Whether to return all job information or just part of it.
```

Comment:
Possibly reword to something that doesn't imply it's boolean (even though it is at the moment).
Something like:

"The information to retrieve about each job"

? (The REST API wording isn't terribly helpful here.)

Reply:
Done